Components/GridColumn.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component, PropTypes } from 'react';
+
+export default class GridColumn extends Component {
+  static propTypes = {
+    children: PropTypes.any,
+    col: PropTypes.string,
+  };
+
+  render() {
+    return <div className={`GridColumn-col${this.props.col}`}>{this.props.children}</div>
+  }
+}
```

Comment:
Because it's being used in other components. It's used on `Footer` as `<Grid.Column>`. I initially had `div className="Grid-column"`, however I was "leaking" `Grid` internal information to the `Footer` component - namely what defines a `Grid-column`. I'd rather hide it this way instead of that alternative, which seems to me as creating a dependency between different components.

If you prefer it that way, I can change it, but that was my reasoning.